docs/articles/configs/toolchains.md

```diff
@@ -87,10 +87,10 @@ namespace BenchmarkDotNet.Samples
         static void Main(string[] args)
         {
             var config = DefaultConfig.Instance
-                .With(Job.Default.With(CoreRuntime.Core21))
-                .With(Job.Default.With(CoreRuntime.Core30))
-                .With(Job.Default.With(ClrRuntime.Net48))
-                .With(Job.Default.With(MonoRuntime.Default));
+                .AddJob(Job.Default.WithRuntime(CoreRuntime.Core21))
```

Comment:
Looks like I missed this one in #2505. We no longer support runtimes older than Core 3.1.

```suggestion
```